rfcs/convergence/native-element-props.md

```diff
@@ -0,0 +1,324 @@
+# RFC: Customizing application of native element props and `ref`
+
+@ecraig12345 @sopranopillow @behowell
+
+## Summary
+
+Instead of always applying native props and `ref` to the root, update `makeMergeProps` to allow setting a different "primary" slot where these props should be applied. This option should be used sparingly: for input components and possibly a few other cases (options for exact criteria discussed below).
```

Comment:
`makeMergeProps` will be deprecated after [as prop simplification RFC](https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/simplify-prop-merging.md)

Reply:
Ah okay, I was wondering about that after trying to copy some of the Accordion code in Input and noticing makeMergeProps was gone. Where would this go instead?

Reply:
Right now this is done manually on state construction as you can verify on Accordion code.  Biggest change from [as prop simplification RFC](https://github.com/microsoft/fluentui/blob/master/rfcs/convergence/simplify-prop-merging.md) is the usage of a more explicit approach for state declaration, it becomes dev responsibility to pass `ref` and other props to the proper `slot`.